api/orders.js

```diff
@@ -99,8 +99,8 @@ function getOrders(request, response, next) {
   function getParsedOrders(offers) {
     return _.reduce(offers, function(orders, off) {
       var sequence = off.seq;
-      var passive = off.flags === ripple.Remote.flags.offer.Passive;
-      var type = off.flags === ripple.Remote.flags.offer.Sell ? 'sell' : 'buy'
+      var passive = off.flags & ripple.Remote.flags.offer.Passive ? true : false;
```

Comment:
Could be slightly type-cleaner to do `off.flags & ripple.Remote.flags.offer.Passive !== 0`